lib/auth/auth.go

```diff
@@ -1749,16 +1750,14 @@ func (a *Server) generateUserCert(req certRequest) (*proto.Certs, error) {
 	}
 
 	pinnedIP := ""
-	if req.checker.PinSourceIP() {
-		if req.clientIP == "" && req.sourceIP == "" {
-			// TODO(anton): make sure all upstream callers provide clientIP and make this into hard error instead of warning.
+	if req.checker.PinSourceIP() || req.pinIP {
```

Comment:
Should we then rename `PinSourceIP` to `PinLoginIP` as well so it's consistent everywhere?

Reply:
You mean just function name on the checker? Because I wouldn't want to change actual role config option `pin_source_ip` 🤔

Reply:
Yeah, we don't want to change role option. It's fine, we can keep this as-is.